inc/sendmail.php

```diff
@@ -23,8 +26,13 @@
 
 
     if ($is_local_environment) {
-        // show the sent email on local
-        echo $html;
+        echo("<script type='text/javascript'>
```

Comment:
Can you assign the success script into a variable (before the if statement) so that we don't have duplicate code below? The variable can be used here and on line 64.